internal/eslint-config/package.json

```
@@ -1,36 +1,34 @@
 {
   "name": "@element-plus/eslint-config",
   "version": "0.0.1",
+  "type": "module",
   "description": "ESLint Config",
   "license": "MIT",
   "files": [
-    "index.js"
+    "index.mjs"
   ],
-  "main": "index.js",
+  "main": "index.mjs",
```

Because now the package is esm friendly by enabling `type: "module"` we can switch `index.mjs` => `index.js` i think.

```suggestion
    "index.js"
],
  "main": "index.js",
```